File: 240206/특정 수와 근접한 합/sum-close-to-particular-number.py
import sys
n,s=map(int,input().split())

num=list(map(int,input().split()))

ans=sys.maxsize

for i in range(n-1):
    for j in range(i+1,n):
        cur=abs(s-(sum(num)-num[i]-num[j]))
        ans=min(ans,cur)

print(ans)

File: 260109/특정 수와 근접한 합/sum-close-to-particular-number.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const [n, s] = input[0].split(' ').map(Number);
const arr = input[1].split(' ').map(Number);

// Please Write your code here.

let sum = 0;
arr.forEach((n)=>sum+=n);

let ans = Number.MAX_SAFE_INTEGER;

for(let i=0; i<n; i++){
    for(let j=i+1; j<n; j++){
        let cnt = sum-(arr[i]+arr[j]);
        ans = Math.min(ans, cnt>s?cnt-s:s-cnt);
    }
}

console.log(ans)